### DISCUSSION TITLE
It was banned by GFW :(

### DISCUSSION THREAD

**Discussion by MotWnb** [ztjhz/BetterChatGPT#265] [General]
“bettergpt.chat" is inaccessible without VPN in China

**Comment by MotWnb** [comment]
:(((((((((((((

**Comment by ztjhz** [comment]
You can download the desktop app [here](https://github.com/ztjhz/BetterChatGPT/releases) or deploy your own Better ChatGPT.

**Reply by MotWnb** [reply]
oh thanks
can i install it on my USB?

**Reply by MotWnb** [reply]
Does it work?

**Reply by ztjhz** [reply]
Yes
